one-to-one-scheduler/one-to-one-scheduler.gs

```diff
@@ -355,44 +393,31 @@ function addAttendees(calendarId, meetId, emails) {
 }
 
 function sendInvitesToStudents() {
-  let calendar = getCalendar();
-  if (!calendar) {
-    showInfo(TITLE_ERROR, MSG_CALENDAR_NOT_FOUND);
-    return;
-  }
-
-  if (!confirmCalendar(calendar)) {
-    showInfo(TITLE_ERROR, MSG_CALENDAR_REJECTED);
-    return;
-  }
-
-  let activeSheet = SpreadsheetApp.getActiveSheet();
-  let oneToOneData = get1to1Data(activeSheet);
-  if (!oneToOneData) {
-    showInfo(TITLE_ERROR, MSG_COULD_NOT_FETCH_DATA);
+  let setupResult = initSetup();
+  if (!setupResult.success) {
     return;
   }
 
   let errorRows = [];
   let skippedRows = [];
   let successRows = [];
-  for (let rowIndex = 0; rowIndex < oneToOneData.length; ++rowIndex) {
+  for (let rowIndex = 0; rowIndex < setupResult.oneToOneData.length; ++rowIndex) {
```

Comment:
Please use destructuring like below. That way you don't need to edit the usages.
```js
let { calendar, activeSheet, oneToOneData } = setupResult;
```